sequential/quadtree.cpp

```diff
@@ -0,0 +1,248 @@
+#include<bits/stdc++.h>
+
+using namespace std;
+
+// Create the definition of a point for the QuadTree
+struct Point {
+    // Each point is defined as (x,y)
+    int x;
+    int y;
+
+    Point(int xVal = 0, int yVal = 0){
+        x = xVal;
+        y = yVal;
+    }
+};
+
+// The Nodes of a QuadTree
+struct Node {
+    Point NodePos;
+    int data;
```

Comment:
Why not combine both Node and Point structs? Isn't there a 1:1 mapping between them?